lib/verifier/hathifiles_database.rb

```diff
@@ -39,7 +39,8 @@ def verify_hathifiles_database_log
         next unless File.exist?(d.path)
 
         if !self.class.has_log?(hathifile: d.path)
-          error message: "missing hf_log: no entry for #{d.path}"
+          recent = Services[:database][:hf_log].order(Sequel.desc(:time)).limit(5).all
```

Comment:
All is well as long as `recent` knows how to stringify itself properly.

Reply:
Should always be an Array or raise an exception if something goes horribly awry. So I'm going to leave it alone for now.